cmd/create/cmd.go

```diff
@@ -59,4 +59,9 @@ func init() {
 	arguments.AddProfileFlag(flags)
 	arguments.AddRegionFlag(flags)
 	confirm.AddFlag(flags)
+
+	accountroles.Cmd.SetHelpFunc(func(command *cobra.Command, strings []string) {
```

Comment:
i dont see region in account roles create . why do you want to do it in this file anyways

Reply:
AAH I see this is help